draft-openid-ipsie-sl1-profile.md

```diff
@@ -102,39 +100,39 @@ This document uses the term "Application" to refer to the "Relying Party" in [Op
+* SHOULD use DNSSEC to protect against DNS spoofing attacks that can lead to the issuance of rogue domain-validated TLS certificates; and
+* MUST perform a TLS server certificate check, as per [RFC9525].
 
 ### Requirements for endpoints not used by web browsers
 
 For server-to-server communication endpoints that are not used by web browsers, the following requirements apply:
 
-* When using TLS 1.2, servers shall only permit the cipher suites recommended in [BCP195];
-* When using TLS 1.2, clients should only permit the cipher suites recommended in [BCP195].
+* When using TLS 1.2, servers MUST only permit the cipher suites recommended in [BCP195];
+* When using TLS 1.2, clients SHOULD only permit the cipher suites recommended in [BCP195].
 
 ### Requirements for endpoints used by web browsers
 
 For endpoints that are used by web browsers, the following additional requirements apply:
 
-* Servers shall use methods to ensure that connections cannot be downgraded using TLS stripping attacks. A preloaded [preload] HTTP Strict Transport Security policy [RFC6797] can be used for this purpose. Some top-level domains, like .bank and .insurance, have set such a policy and therefore protect all second-level domains below them.
-* When using TLS 1.2, servers shall only use cipher suites allowed in [BCP195].
-* Servers shall not support [CORS] for the authorization endpoint, as clients must perform an HTTP redirect rather than access this endpoint directly.
+* Servers MUST use methods to ensure that connections cannot be downgraded using TLS stripping attacks. A preloaded [preload] HTTP Strict Transport Security policy [RFC6797] can be used for this purpose. Some top-level domains, like .bank and .insurance, have set such a policy and therefore protect all second-level domains below them.
+* When using TLS 1.2, servers MUST only use cipher suites allowed in [BCP195].
+* Servers MUST NOT support [CORS] for the authorization endpoint, as clients must perform an HTTP redirect rather than access this endpoint directly.
 
 
 ## Cryptography and Secrets
 
 The following requirements apply to cryptographic operations and secrets:
 
-* Authorization servers, clients, and resource servers when creating or processing JWTs shall:
+* Authorization servers, clients, and resource servers when creating or processing JWTs MUST:
   * adhere to [RFC8725];
   * use PS256, ES256, or EdDSA (using the Ed25519 variant) algorithms; and
-  * not use or accept the `none` algorithm.
-* RSA keys shall have a minimum length of 2048 bits.
-* Elliptic curve keys shall have a minimum length of 224 bits.
-* Credentials not intended for handling by end-users (e.g., access tokens, refresh tokens, authorization codes, etc.) shall be created with at least 128 bits of entropy such that an attacker correctly guessing the value is computationally infeasible ({{Section 10.10 of RFC6749}}).
+  * NOT use or accept the `none` algorithm.
```

Comment:
This NOT is separated from the MUST on line 129 - is this acceptable and clear for readers?  Consider moving the MUST on 129 to each subordinate bullet point for more clarity.

Reply:
yeah I agree it would read better that way, I'll add it to each bullet